styles/logo-file-icons.less

```diff
@@ -184,77 +174,26 @@
 
   [data-name$=".scala"]:before { .file-icon(scala); }
 
-  span[data-name=".npmrc"]:before,
-  span[data-name="npm-debug.log"]:before,
-  span[data-name=".npmignore"]:before,
-  div[data-name$="package.json"]:before,
-  span[data-name$="package.json"]:before { .file-icon(npm); }
+  [data-name=".npmrc"]:before,
+  [data-name="npm-debug.log"]:before,
+  [data-name=".npmignore"]:before,
+  [data-name="package.json"]:before { .file-icon(npm); }
```

Comment:
hmm you're sure this is working? I have added the `span` to the selector to give it a higher specificity, since `[data-name="package.json"]` was "overwritten" by `[data-name$=".json"]`.